spec/controllers/api/events_controller_spec.rb

```diff
@@ -87,7 +87,7 @@ def overridden_params(new_params)
 
         it "should return 422" do
           allow(subject).to receive(:create_params)
-            .and_raise(ActionDispatch::ParamsParser::ParseError.new('test', 'test'))
```

Comment:
`ActionDispatch::ParamsParser::ParseError` will be depecrated after 5.0 
See : https://guides.rubyonrails.org/5_0_release_notes.html
https://stackoverflow.com/questions/47942780/migrating-from-rails-4-to-rails-5-using-actiondispatchrequest-parameter-parse

TODO: Update `application.rb` to use middleware by instead of sending as strings send over the class i.e. `CatchJsonApiParseErrors` vs `"CatchApiJsonParseErrors"`